template/src/pages/VideoCall.tsx

```diff
@@ -372,7 +389,8 @@ const VideoCall: React.FC = () => {
                   : ChannelProfile.Communication,
               }}>
               <RtcConfigure>
-                <DeviceConfigure userRole={rtcProps.role}>
+                <DeviceConfigure
+                  userRole={rtcProps?.role || ClientRole.Audience}>
```

Comment:
if you need it as audience for if no role, why no just default initialize the rtcProps to ClientRole.Audience instead of ClientRole.Broadcaster